config/application.rb

```diff
@@ -26,7 +26,6 @@ module TeacherVacancyService
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-    config.autoloader = :classic
```

Comment:
❤️

Reply:
So long, classic autoloader. Thanks for all the memories! 😢